geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientHealthMonitor.java

```diff
@@ -100,7 +107,7 @@
   /**
    * The interval between client monitor iterations
    */
-  final protected static long CLIENT_MONITOR_INTERVAL = 1000;
+  final private static long DEFAULT_CLIENT_MONITOR_INTERVAL = 1000;
```

Comment:
I like to postfix the field with "IN_MILLIS" or "IN_SECONDS". 1000 does not really mean anything to me without a unit